### PR TITLE
TELCODOCS-429 removing TP from Pod-level bonding for secondary networks CNF-3877 (TELCODOCS-429)

### PR DESCRIPTION
**Pull request by kquinn1204**
[TELCODOCS-429]: [CNF-3877](https://issues.redhat.com//browse/CNF-3877) Promote bond CNI to GA

Version(s):
4.12 RN

Issue:
https://issues.redhat.com/browse/TELCODOCS-429

see also PR https://github.com/openshift/openshift-docs/pull/47172 for context
Link to docs preview: http://file.emea.redhat.com/kquinn/TELCODOCS-429-410/release_notes/ocp-4-10-release-notes.html#ocp-4-10-technology-preview

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by kquinn1204** [comment]
@mmirecki @cgoncalves I raised this to update the TP and remove the note "The current functionality of Bond CNI..." Franck asked me to update the docs as I assume fix filtered through in z stream . Can you give this a LGTM please

**Comment by cgoncalves** [comment]
/lgtm

Release noting which z-stream feature support levels changed is definitely much better, thanks!

**Comment by kquinn1204** [comment]
@kalexand-rh I have addressed your concerns about the footnote and removed for both pod level bonding and the Web Terminal Operator in this PR. I followed the azure example and put in a link to the correct errata description.

**Comment by kquinn1204** [comment]
@JoeAldinger Hopefully this can be merged

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -3369,6 +3360,14 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.10.30 --pullspecs
 ----
 
+[id="ocp-4-10-30-features"]
+==== Features
+
+[id="ocp-4-10-30-pod-level-bonding-ga"]
+===== General availability of pod-level bonding for secondary networks
+
+With this update, link:https://docs.openshift.com/container-platform/4.10/networking/hardware_networks/using-pod-level-bonding.html[*Using pod-level bonding*] is now generally available.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-10-release-notes.adoc:3369]: This should be an xref rather than a link.

**Review by jeana-redhat — COMMENTED** [review]
This is very neatly done. The only issue I see is the link in the .z section

**Pull request merged**